vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -4,7 +4,12 @@ sidebarDepth: 2
 
 # Getting Started with Entando
 
+::: warning
+**Mac:** Support for the Apple M1 platform is a work in progress and not offered by Entando 6.
```

Comment:
I had to stare at it for a bit, "not offered" doesn't feel quite right. We do offer enterprise support for the Entando platform in production which is a different thing altogether.  Maybe "Entando 6 is not currently compatible with the Apple M1 ARM64 architecture found in some newer Macs."

Reply:
I think this is closer to your release note which I do like. This doesn't promise too much. ;)

Reply:
Good call! You highlight a distinction that hadn't occurred to me - there are many flavors and stages of implementation across different sectors. Support in production is far different from a consumer use case where a customer is struggling with local installation and doesn't have an FAE resource.